patches/0002-Add-Mariner-support-with-FIPS-dependency-flag.patch

```diff
@@ -28,7 +28,7 @@ index a79ec02..106f1d0 100644
  -}}
  {{ if is_alpine then ( -}}
  FROM alpine:{{ alpine_version }}
-@@ -14,6 +24,27 @@ RUN apk add --no-cache ca-certificates
+@@ -14,6 +24,29 @@ RUN apk add --no-cache ca-certificates
  # - https://github.com/golang/go/blob/go1.9.1/src/net/conf.go#L194-L275
```

Comment:
Know this isn't changed but why is go 1.9.1 being used? That seems odd at a glance. Know it's a tag, not the actual go release but still seems odd.

Reply:
It looks like this comment was added back when 1.9.1 was in support, so this is just a fairly old pinned link to explain why that `echo 'hosts: files dns' > /etc/nsswitch.conf` line is there on Alpine.
https://github.com/docker-library/golang/commit/43f9c69e1bcb068bdb6883cd0fb2339e9b197510 https://github.com/docker-library/golang/pull/217